app/controllers/queue_items_controller.rb

```diff
@@ -0,0 +1,34 @@
+class QueueItemsController < ApplicationController
+  before_action :require_user
+
+  def index
+    @queue_items = current_user.queue_items.order(:position)
```

Comment:
This is not bad. But I would suggest to order `#queue_items` right in `has_many :queue_items` because it will be ordered everywhere else when you call it, however this applies only to this controller and the corresponding action.